rskj-core/src/main/java/co/rsk/config/BridgeConstants.java

```diff
@@ -86,6 +86,8 @@ public abstract class BridgeConstants {
 
     protected int numberOfBlocksBetweenPegouts;
 
+    protected long btcHeightWhenPegoutTxIndexActivates;
```

Comment:
Not a big deal at all, just nitpicking. But an `int` value should probably do it considering the current bitcoin heights. What do you think?